sound/soc/sdca/sdca_device.c

```diff
@@ -22,3 +23,26 @@ void sdca_lookup_interface_revision(struct sdw_slave *slave)
 				 &slave->sdca_interface_revision);
 }
 EXPORT_SYMBOL_NS(sdca_lookup_interface_revision, SND_SOC_SDCA);
+
+bool sdca_device_quirk_match(struct sdw_slave *slave, enum sdca_quirk quirk)
+{
+	struct sdw_slave_id *id = &slave->id;
+
+	switch (quirk) {
+	case SDCA_QUIRKS_RT712_VB:
+		if ((slave->sdca_interface_revision > 0x0604) &&
```

Comment:
Maybe >= 0x1000? As far as I know, the function type definition changed from sdca 0.6 to 1.0.

Reply:
@bardliao @shumingfan I could see this value in the ASL file:
Package (2) {"mipi-sdw-sdca-interface-revision", 0x0801}, // Integer: SDCA 0.8r01

Can I use it as an indicator for VB, or just keep the > 0x0604 that was used for VA?